test/blackbox-tests/test-cases/formatting/format-dune-file.t/run.t

```diff
@@ -188,3 +188,47 @@ Non 0 error code:
   File "", line 2, characters 0-0:
   Error: unclosed parenthesis at end of input
   1
+
+Using the built-in action.
```

Comment:
Maybe add a test demonstraating how the action fails when given an invalid dune file?